### ISSUE TITLE
App wrongly shows "RISIKO-ERMITTLUNG AKTIV" in case Standortermittlung is not activated

### ISSUE THREAD

**Issue by pahl34** [corona-warn-app/cwa-app-android#561]
This remark by @KaiRoesner is a very important one and thus I open a separate issue for it:

-----

_Another remaining question is whether the exposure detection works or not with GPS switched off. Because (at least on my Moto G6 phone) the app still says "Risiko-Bewertung aktiv" even when I switch GPS off (but leave Bluetooth activated)._

_Originally posted by @KaiRoesner in https://github.com/corona-warn-app/cwa-app-android/issues/519#issuecomment-644790565_

-----

As already broadly discussed here and in (social) media, the android app has a confusing behavior regarding the necessity of activating "Standortermittlung". However, a huge problem in usability of the app that clearly has not been addressed prominently enough is the fact that apparently the app does not work in case "Standortermittlung" is deactivated, but the user interface *does not indicate* it.

This article from Spiegel clearly says that the app will not work in case "Standortermittlung" is not active:

https://www.spiegel.de/netzwelt/apps/corona-warn-app-wieso-muessen-android-nutzer-den-standortzugriff-aktivieren-a-7f372aea-25e9-49f7-9ae3-9568cf526c04

If this is true, then it is a big problem, that the "RISIKO-ERMILLUNG AKTIV" indicator still shows 'active' in this situation. I fear it will lead to thousands of installed apps that are not working unnoticed by the users. This would mean that the whole purpose of the app might not be fulfilled as intended and it might well lead to failing to save lives.

Many users don't like to keep "Standortermittlung" active all the time. If deactivated, android will show a notification/warning that can just be swiped and it is gone. This will happen and people will believe that the app works without "Standortermittlung" because the app itself does not indicate any problem.

Please be aware that usability is extremely important here. We cannot expect a major part of the users to be technical experts, many might even not have any technical background at all.

In case Spiegel should be wrong and the app still works even with "Standortermittlung" being deactivated during usage of the app, then this issue could be closed - however this would also be a big problem and a clear and broadly visible statement for everyone would be required in this case.

**Comment by aurisnoctis** [comment]
>  If deactivated, android will show a notification/warning that can just be swiped and it is gone.

I don't get any warning when I turn off location in the Android settings (balloon icon) - no warning from the Corona-Warn-App nor from Android (6.0.1). And the Corona-Warn-App still shows that tracing is active.

People tried to convince me that as long as the app doesn't show any problem everything works OK. I fear otherwise: lots of useless installations with meaningless placeholder beacons.

See also #491

**Comment by aurisnoctis** [comment]
For an asnwer see specifically https://github.com/corona-warn-app/cwa-app-android/issues/491#issuecomment-644710288
If I understand correctly, the answer is the exposure detection does not work with location switched off (specifically GPS may be switched off, but not location overall, which also includes the scanning of wifi and bluetooth to determine location). If the above is correct, the app should warn that tracing is inactive when location is off and you can join issue #491 to demand a fix.

**Comment by pahl34** [comment]
I agree that my issue entry is a duplicate of #491 .
I had tried to search for potential duplicates before but somehow I didn't catch this one.
Thanks for pointing out.

I'd like to mention again that I think that this is a very severe problem, as it will very likely lead to thousands of non-active app instances, **that remain unnoticed** by the users. Thus, I suggest to work on a fix for this bug with a very high priority.

Closing the issue now, as there already exists a duplicate with deeper discussion already (#491 )
